ssntp/client.go

```diff
@@ -368,7 +368,7 @@ func (client *Client) sendCommand(cmd Command, payload []byte, trace *TraceConfi
 	client.status.Lock()
 	if client.status.status == ssntpClosed {
 		client.status.Unlock()
-		return -1, fmt.Errorf("Client not connected")
+		return -1, fmt.Errorf("sendCommand: Client not connected")
 	}
 	client.status.Unlock()
 
```

Comment:
I guess we can always change it later if we decide we want that extra level of detail.  So, make it say whatever you want.